content/post/activity_hours.md

```diff
@@ -57,24 +45,25 @@ It normally takes less than 5 minutes to get the cluster running and fully opera
 
 The next step is to download the datasets. data.world has a size limit which prevents from uploading files bigger than 50MB, so we decided to upload ours to Google Drive for ease of use.
 
-<pre class="verbatim">
+```
```

Comment:
I think this is a first iteration @vmarkovtsev 
I think it can be done in the styling part https://github.com/src-d/blog/pull/121